# Hi there, I'm Martin ðŸ‘‹

## About Me
I'm passionate about developing and applying data analysis techniques, machine learning, and computational modeling to all types of data, though most of my professinal experience is from the field of biology.

## ðŸ”§ Technologies & Tools
- **Programming Languages:**
  - Python
  - R
    
- **Machine Learning / AI:** 
  - Classical machine learning (scikit-learn)
  - Deep learning and neural networks (PyTorch)
  - Probabilistic models (Pyro)
    - [Project](https://github.com/mengqvist/data_analysis_sivia/): Solving probabilistic problems from the "Data Analysis - A Bayesian Tutorial" textbook with Pyro.
    - [Project](https://github.com/mengqvist/data_analysis_mbml/): Solving probabilistic problems from the "Model-Based Machine Learning" textbook with Pyro.

- **Modeling / Optimization:**
  - [Project](https://github.com/mengqvist/phonetic-distance): Modeling English words to find which sound similar/different, including optimization of parameters.
      
- **Bioinformatics:**
  - Structure analysis (AlphaFold2, Rosetta, APBS, PyMol)
  - Sequence analysis (HHblits, CD-HIT, HMMER, BLAST, ESM2, etc.)
    - [Project](https://github.com/EngqvistLab/yeast_xylanase): CAZyme prediction in Ascomycetes yeast genomes
    - [Project](https://github.com/EngqvistLab/analyze_1.1.3.15): Experimental exploration of the enzyme class EC 1.1.3.15

- **Data Analysis & Mining:**
  - Web scraping (selenium)
  - Dataset analysis and integration (pandas)

- **Other:**
  - Git
  - Computational pipeline development
    
## ðŸš€ Selected Publications
- **Protein Substrate & Activity Prediction:**
  - [Deep learning-based kcat prediction enables improved enzyme-constrained model reconstruction](https://doi.org/10.1038/s41929-022-00798-z)
  - [A general model to predict small molecule substrates of enzymes based on machine and deep learning](https://doi.org/10.1371%2Fjournal.pcbi.1009446)
  - [Deep learning allows genome-scale prediction of Michaelis constants from structural features](https://doi.org/10.1371/journal.pbio.3001402)
    
- **Enzyme Engineering & Directed Evolution:**
  - [Expanding functional protein sequence spaces using generative adversarial networks](https://doi.org/10.1038/s42256-021-00310-5)
  - [Chiral Alcohols from Alkenes and Water: Directed Evolution of a Styrene Hydratase](https://doi.org/10.1002/anie.202215093)
  - [Directed Evolution of (R)-2-Hydroxyglutarate Dehydrogenase Improves 2-Oxoadipate Reduction by 2 Orders of Magnitude](https://pubs.acs.org/doi/10.1021/acssynbio.2c00162)
  - [Modeling-Assisted Design of Thermostable Benzaldehyde Lyases from Rhodococcus erythropolis for Continuous Production of Î±-Hydroxy Ketones](https://doi.org/10.1002/cbic.202100468)
  - [Directed Evolution of Gloeobacter violaceus Rhodopsin Spectral Properties](https://doi.org/10.1016/j.jmb.2014.06.015)
  - [Archaerhodopsin variants with enhanced voltage-sensitive fluorescence in mammalian and Caenorhabditis elegans neurons](https://www.nature.com/articles/ncomms5894)
  - [Directed evolution of a far-red fluorescent rhodopsin](https://doi.org/10.1073/pnas.1413987111)
    
- **Next-Generation Sequencing Data Processing:**
  - [Ribonucleotides incorporated by the yeast mitochondrial DNA polymerase are not repaired](https://doi.org/10.1073/pnas.1713085114)
  - [Nucleotide pools dictate the identity and frequency of ribonucleotide incorporation in mitochondrial DNA](https://doi.org/10.1371/journal.pgen.1006628)
      
## âœ¨ Other Work
You will find more of my previous work at my old research groups [GitHub page](https://github.com/EngqvistLab)

## ðŸ“« How to Reach Me
- [LinkedIn](https://www.linkedin.com/in/engqvist1729)
- [Google Scholar](https://scholar.google.com/citations?user=ZTyba9cAAAAJ&hl=en)


![Profile views](https://komarev.com/ghpvc/?username=mengqvist&color=green)
